capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/CameraFragmentImpl.java

```diff
@@ -752,6 +760,38 @@ private void bindViews(final View view) {
         mUnsupportedQRCodeDetectedPopupContainer = view.findViewById(
                 R.id.gc_unsupported_qrcode_detected_popup_container);
         mImageStack = view.findViewById(R.id.gc_image_stack);
+        topAdapterInjectedViewContainer = view.findViewById(R.id.gc_navigation_top_bar);
+
+    }
+
+    private void setTopBarInjectedViewContainer() {
+        if (GiniCapture.hasInstance()) {
+            topAdapterInjectedViewContainer.setInjectedViewAdapter(GiniCapture.getInstance().getNavigationBarTopAdapter());
+
+            if (topAdapterInjectedViewContainer.getInjectedViewAdapter() == null)
+                return;
+
+            if (mFragment.getActivity() == null)
+                return;
+
```

Comment:
[Request] Back button click listener is missing.

Reply:
Done